### ISSUE TITLE
`cargo install grcov` fails due to modern `tera` versions

### ISSUE THREAD

**Issue by A0lson** [mozilla/grcov#700]
The `tera` dependency in `Cargo.toml` isn't properly specified and as a result, all recent `grcov` releases fail to install using something like `cargo install grcov`.

For example, with a Rust 1.52 toolchain (`1.52.0-x86_64-unknown-linux-gnu`),  attempting  `cargo install grcov` results in:
```
error[E0107]: missing generics for struct `tera::Context`
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/grcov-0.8.2/src/html.rs:211:22
    |
211 | fn make_context() -> Context {
    |                      ^^^^^^^ expected 1 generic argument
    |
note: struct defined here, with 1 generic parameter: `S`
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/tera-1.14.0/src/context.rs:57:12
    |
57  | pub struct Context<S: ContextSafety> {
    |            ^^^^^^^ -
help: add missing generic argument
    |
211 | fn make_context() -> Context<S> {
    |                      ~~~~~~~~~~

For more information about this error, try `rustc --explain E0107`.
error: could not compile `grcov` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `grcov v0.8.2`, intermediate artifacts can be found at `/tmp/cargo-installngIiQg`

Caused by:
  build failed

```

Either the `grcov` code should be updated or at least the dependency tightened to use older `tera` versions that do not have this problem.

Using `cargo install --locked` isn't very nice as at least one package is already "yanked".

**Comment by akanawati** [comment]
Same issue.  This breaks with stable as well (1.56 today).

**Comment by marco-c** [comment]
Released 0.8.3 with a fix.
